requirements.txt

```diff
@@ -10,4 +10,5 @@ jax==0.2.9
 jaxlib==0.1.59
 hilbertcurve==1.0.5
 networkx==2.5
-cvxpylayers==0.1.5 # this depends on jax -- upgrade when jax / jaxlib are upgraded
+cvxpylayers==0.1.5  # this depends on jax -- upgrade when jax / jaxlib are upgraded
+tqdm                # progress bars
```

Comment:
Would be good to pin this version to be safe

Reply:
+1 on pinning this before merging.

Reply:
Addressed in https://github.com/proteneer/timemachine/pull/442/commits/06562616bffe6e0a0c64e04d1697c6efd4221235